components/Onboarding/Sections/Newsletter.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Query } from 'react-apollo'
+import gql from 'graphql-tag'
+import { css } from 'glamor'
+
+import Section from '../Section'
+import NewsletterSubscriptions from '../../Account/NewsletterSubscriptions'
+
+import { Interaction } from '@project-r/styleguide'
+
+const { P } = Interaction
+
+const styles = {
+  p: css({
+    marginBottom: 20
+  })
+}
+
+const NEWSLETTER_CONSENTS = gql`
```

Comment:
Why are you using consents here? IMO should use `me.newsletterSettings` because we auto subscribe on purchase.

Reply:
Used soley to evaluate if section checkmark ("tick") should be set.